x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.test.ts

```diff
@@ -586,6 +586,36 @@ const ERROR_NAMESPACE_SPECIFIED = 'Spaces currently determines the namespaces';
       });
     });
 
+    describe('#bulkDelete', () => {
+      test(`throws error if options.namespace is specified`, async () => {
+        const { client } = createSpacesSavedObjectsClient();
+
+        await expect(
+          // @ts-expect-error
+          client.bulkDelete(null, { namespace: 'bar' })
+        ).rejects.toThrow(ERROR_NAMESPACE_SPECIFIED);
+      });
+
+      test(`supplements options with the current namespace`, async () => {
+        const { client, baseClient } = createSpacesSavedObjectsClient();
+        const expectedReturnValue = { statuses: [{ id: 'id', type: 'type', success: true }] };
+        baseClient.bulkDelete.mockReturnValue(Promise.resolve(expectedReturnValue));
+
+        const actualReturnValue = await client.bulkDelete([{ id: 'id', type: 'foo' }]);
```

Comment:
NIT: call `bulkDelete` with the `options` parameter (e.g `{ force: true }`) to make sure that the provided options and the added `namespace` one are properly merged.

Reply:
The spaces saved objects client declares all `options` as `SavedObjectsBaseOptions`, even though the base client (core's base SavedObjectsClient) types some differently.

For example, core's base client bulkUpdate [options](https://github.com/elastic/kibana/blob/main/x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.ts#L276) are
```
export interface SavedObjectsBulkUpdateOptions extends SavedObjectsBaseOptions {
  /** The Elasticsearch Refresh setting for this operation */
  refresh?: MutatingOperationRefreshSetting;
}
```
while the spaces client decares the [bulkUpdate options](https://github.com/elastic/kibana/blob/main/x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.ts#L276) as `SavedObjectsBaseOptions`:
```
export interface SavedObjectsBaseOptions {
  /** Specify the namespace for this operation */
  namespace?: string;
}
```

TBH, I'm not sure why, exactly, the spaces saved objects client has subtle type differences when it comes to the options. Since `force` is optional anyway, I don't see any harm in changing the type for `bulkDelete`.